### PR TITLE
Add new scenario dropdown; remove scenario tabs

### PR DESCRIPTION
**Pull request by arottersman**
## Overview

The new wireframes remove the scenario tabs and replace them with a scenario dropdown. 

**Wireframes**
screen shot 2017-05-10 at 4 09 19 pm

Connects #1865 

### Demo
TR-55
screen shot 2017-05-10 at 4 13 54 pm

GWLFE
screen shot 2017-05-10 at 4 15 21 pm

al4wj5xdxc

### Notes

- The ". . ." more options button is temporary. Jeff is still working on where to put the Share and Duplicate actions, and what their icons should look like.
- The font sizes for the scenario and project titles don't quite match the wireframes. I wanted to use one of the consolidated font-size vars @jfrankl created, and not add to the mess
- The styling is messed up in the toolbar: the scenario title doesn't line up with the buttons, and the buttons on narrow viewports fall off the line. I think this should be addressed in #1867
- I didn't add the popover:
screen shot 2017-05-10 at 3 53 13 pm

There's a title for now:
screen shot 2017-05-10 at 3 50 34 pm 2

I'll make an issue for it

- This is messed up, but the whole top part of that view will be removed with #1873 
screen shot 2017-05-10 at 4 15 36 pm

## Testing Instructions

* Pull this branch
 * `./scripts/bundle.sh` `./scripts/bundle.sh --vendor`
* Create a TR-55 project
    - Test initial current conditions view is not a dropdown
    - Add a scenario with the "+ Add changes button"
    - Test each action on the scenarios
* Complete the above for Mapshed
* Test a public project that you don't own shows no action icons on its scenarios

### CODE REVIEW

File: src/mmw/js/src/modeling/views.js [hunk 1]
```diff
@@ -266,132 +269,173 @@ var ProjectMenuView = Marionette.ItemView.extend({
...
-        this.$el.toggleClass('active', this.model.get('active'));
+        'change': 'render',
     },
 
     ui: {
-        share: '[data-action="share"]',
+        selectScenario: '[data-action="select"]',
         destroyConfirm: '[data-action="delete"]',
         rename: '[data-action="rename"]',
-        print: '[data-action="print"]',
```
Comment (arottersman) [inline src/mmw/js/src/modeling/views.js]: I couldn't find this in the ui.

Comment (mmcfarland) [inline src/mmw/js/src/modeling/views.js]: Looks vestigial from https://github.com/WikiWatershed/model-my-watershed/commit/c0d911be34625d7d9529062f7b985991efb0aaf4

There may be a few more references as well.

**Comment by caseycesari** [comment]
This may be something to take care of in a follow-up issue, but the "rename scenario" UI needs a little work:

image

image

**Comment by arottersman** [comment]
Sadly, IE11 analyze and model are broken on staging as well:
screen shot 2017-05-12 at 9 58 35 am

**Comment by caseycesari** [comment]
Good catch. I should have checked that first.

**Comment by arottersman** [comment]
@azavea-bot rebuild

**Comment by arottersman** [comment]
Thanks, Casey!

**Pull request merged**